src/test/java/com/salesforce/storm/spout/dynamic/DynamicSpoutTest.java

```diff
@@ -1002,10 +1001,10 @@ public void testConsumeWithConsumerGroupOddNumberOfPartitions(final int taskInde
     @Test
     public void testReportErrors() {
         // Define config
-        //final Map<String, Object> config = getDefaultConfig("ConsumerIdPrefix", "StreamId");
-        final Map<String, Object> config = new HashMap<>();
+        Map<String, Object> config = new HashMap<>();
         config.put(SpoutConfig.VIRTUAL_SPOUT_ID_PREFIX, "ConsumerIdPrefix");
         config.put(SpoutConfig.PERSISTENCE_ADAPTER_CLASS, InMemoryPersistenceAdapter.class.getName());
+        config = SpoutConfig.setDefaults(config);
```

Comment:
Elsewhere we seem to normally set defaults when creating the map and then putting onto it.